samples/angular/src/app/routing/jss-route-resolver.service.ts

```diff
@@ -13,7 +13,7 @@ export class JssRouteResolver implements Resolve<JssState> {
 
   resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<JssState> {
     // in experience editor, we need to reload to avoid confusing the editor ribbon
-    if (isExperienceEditorActive() && window) {
+    if (isEditorActive() && window) {
```

Comment:
Should we check both for Horizon and EE? Should it be executed only in Experience Editor? As I can see line 15.
In another case need to update comment: experience editor -> Sitecore Editor

Reply:
I updated the comment ("experience editor" > "Sitecore editor" and removed "ribbon") to make it editor agnostic. This should be the desired behavior (i.e. best we can do) in both EE and Horizon.